sanic/server.py

```diff
@@ -301,6 +303,9 @@ def connection_made(self, transport):
             self.transport = transport
             self._task = self.loop.create_task(self.connection_task())
             self.recv_buffer = bytearray()
+            self._buffer = memoryview(
+                bytearray(self.app.config.REQUEST_BUFFER_SIZE)
+            )
```

Comment:
This is a waste of RAM. The buffer should be kept as small as possible, not always the maximum permitted size. This makes a difference when handling a lot of parallel requests.

Reply:
I am not sure where this value is coming from, but it is the size limit that `get_buffer` seems to want :thinking: 
```
>> get_buffer sizehint=65536
```

Reply:
The sizehint is hardcoded in libuv (and probably in a similar manner in plain asyncio). We can safely ignore the hint because libuv will then simply call `recv_into` with a smaller buffer. For the sake of simplicity, it might be best to keep our buffer hardcoded to 64 KiB as well, at least until it can be evaluated whether BufferedProtocol offers those performance benefits that we want.